py3-greenlet.spec

```diff
@@ -0,0 +1,4 @@
+### RPM external py3-greenlet 1.1.2
+## IMPORT build-with-pip3
+
+%define pip_name
```

Comment:
@vkuznet can you please remove this line?

Reply:
done